deployment/create_docker_image.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+INDEX_REPO="clangd/llvm-remote-index"
+OUTPUT_NAME="clangd-linux-snapshot.zip"
+IMAGE_NAME="llvm-remote-index-server"
+INDEX_ASSET_PREFIX="llvm-index"
+INDEX_FILE_NAME="llvm.idx"
+SERVER_ASSET_PREFIX="clangd-linux-snapshot"
```

Comment:
this should be `clangd-indexing-tools` (also makes sense to change elsewhere for consistency)